plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/client/GumbyClient.kt

```diff
@@ -206,7 +206,7 @@ class GumbyClient(private val project: Project) {
             .connect { request -> // default connect timeout is 10s
                 val connection = request.connection as HttpURLConnection
                 connection.setFixedLengthStreamingMode(fileToUpload.length())
-                connection.readTimeout = 30 * 60 * 1000 // since pre-signed URL expiration was increased to 30 minutes
+                connection.readTimeout = 35 * 60 * 1000 // since pre-signed URL expiration was increased to 30 minutes
```

Comment:
nit: probably `31` or `32` is better since we don't want user to wait much longer beyond 30 mins since URL will be expired